akka-sample-cqrs-scala/src/main/scala/sample/cqrs/CqrsApp.scala

```diff
@@ -34,31 +44,27 @@ object CqrsApp {
     }
   }
 
-  def startClusterInSameJvm(): Unit = {
-    startCassandraDatabase()
-
-    startNode(2551)
-    startNode(2552)
-  }
-
   def startNode(port: Int): Unit = {
-    val system = ActorSystem("ClusterSystem", config(port))
 
-    // TODO start ClusterSharding
+    val system = ActorSystem("ClusterSystem", config(port))
 
-    if (port == 2551) {
-      createTables(system)
+    val selfRoles = Cluster(system).selfRoles
 
+    if (selfRoles.contains("write-model")) {
```

Comment:
move ShardedSwitchEntity(system).start() here, for clarity and symmetry with the read-model